engines/bastion/app/views/bastion/layouts/assets.html.erb

```diff
@@ -18,7 +18,7 @@
     angular.module('Bastion.features').value('FeatureSettings', angular.fromJson(<%= SETTINGS[:features].nil? ? {} : SETTINGS[:features].to_json.html_safe %>));
     angular.module('Bastion').value('currentLocale', '<%= I18n.locale %>');
     angular.module('Bastion').value('CurrentOrganization', "<%= Organization.current.id if Organization.current %>");
-    angular.module('Bastion').value('contentAccessMode', "<%= Organization.current.try(:content_access_mode) if Organization.current %>");
+    angular.module('Bastion').value('simpleContentAccessEnabled', <%= Organization.current.simple_content_access? if Organization.current %>)
```

Comment:
An extra space in front of <%= Organization and a semi-colon is missing in this statement. Just to be consistent :)
Apart from this the PR looks good.